content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "What is expected from staff engineers"
+date: 2022-12-18
+tags: [
+    "career", "growth"
+]
+---
+
+It's relatively easy to explain expectations for junior, mid, and even senior engineers. But it starts to get much harder at staff+ 
+positions. Career progression broadens the scope of work andresponsibilities. The environment and problems get more ambiguous. It's not easy
```

Comment:
Let's avoid phases like `not easy` - and use the synonyms https://www.thesaurus.com/browse/not%20easy
my personal choice would be: `demanding`